metadata-ws/src/test/resources/features/accession.feature

```diff
@@ -0,0 +1,68 @@
+Feature: accession object
+
+  Scenario Outline: find different objects by accession and version
+    When I request POST /taxonomies with JSON payload:
+    """
+    {
+      "taxonomyId": 9606,
+      "name": "Homo Sapiens"
+    }
+    """
+    Then set the URL to TEST_TAXONOMY
+    When I create a test parameterized study with EGAS0001 for accession, 1 for version, test_study for name, false for deprecated, 0 for releaseDay and TEST_TAXONOMY for taxonomy
```

Comment:
can this be done by payload as well

Reply:
We could, but taxonomy URLs can not be sent directly; need to provide a means to extract from map

Reply:
I think one way could make this work is to have placeholders in the payload and on the step handler side, replace the placeholders with real values.

Reply:
I have checked-in payload changes. If everything looks good i can change in all other places